app/src/main/java/fr/neamar/kiss/broadcast/IncomingCallHandler.java

```diff
@@ -25,6 +25,8 @@ public void onReceive(final Context context, Intent intent) {
                 return;
             }
 
+            contactsProvider.reload();
```

Comment:
Since updating contact is time consuming, I'm curious if you've noticed any change on your call quality?